### PR TITLE
update-CHANGELOG and ver # for dashboards

### PR DESCRIPTION
**Pull request by Kully**

### CODE REVIEW

**Comment by Kully** [comment]
@cldougl

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+
+## [2.0.3] - 2017-03-06
 ## Added
 - dashboards can now be created using the API and uploaded to Plotly. Use `import plotly.dashboard_objs` to be create a dashboard a `Dashboard` object. You can learn more about `Dashboard` objects by running `help(plotly.dashboard_objs.Dashboard)` and `help(plotly.plotly.plotly.dashboard_ops)` for uploading and retrieving dashboards from the cloud.
```
Comment (cldougl) [inline CHANGELOG.md]: @Kully 
- Capitalize 1st word of sentence (Dashboards).
- incomplete sentence: 
>Use `import plotly.dashboard_objs` to be create a dashboard a `Dashboard` object.

**Comment by Kully** [comment]
I'm adding a few bug fixes for the dashboard too

File: plotly/dashboard_objs/dashboard_objs.py [hunk 2]
```diff
@@ -507,7 +448,7 @@ def insert(self, box, side='above', box_id=None):
         box_ids_to_path = self._compute_box_ids()
 
         # doesn't need box_id or side specified for first box
-        if self['layout'] is None:
+        if self['layout'] is None:  # or
```
Comment (cldougl) [inline plotly/dashboard_objs/dashboard_objs.py]: @Kully I don't think I understand the comment here?

Comment (Kully) [inline plotly/dashboard_objs/dashboard_objs.py]: lol, that was mistake. I'll remove

**Comment by Kully** [comment]
@cldougl Okay, a :dancing_men: after this passes?

**Comment by cldougl** [comment]
💃

**Pull request merged**